CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+This release is the last to support [Go 1.22].
+The next release will require at least [Go 1.23].
```

Comment:
As a question for advice, I would like to understand the principle behind our support for Go versions.

As far as I know, the [release policy](https://go.dev/doc/devel/release#policy) of the official Go project indeed only maintains the two latest versions. Is it the same for us?

From another perspective, as an open-source application, the more Go versions we support, the lower the requirements for users, and the larger the audience will be.

So, how do we generally choose which versions to support? Or are there any industry references that we can rely on?

Reply:
Yes, we follow the Go support policy. So we support N and N-1.
That's documented here: https://github.com/open-telemetry/opentelemetry-go?tab=readme-ov-file#compatibility